lisk-dex-electron/src/App.jsx

```diff
@@ -25,8 +25,8 @@ import MarketList from './MarketList';
 import Notification from './Notification';
 import userContext from './context';
 import LeaveWarning from './LeaveWarning';
-import processConfiguration from './config/Configuration';
-import { CryptoAsset, getNumericAssetBalance } from './Utils';
+import createRefinedGlobalConfig from './config/Configuration';
+import { getNumericAssetBalance, GlobalConfiguration, GlobalConfiguration as GC } from './Utils';
```

Comment:
We shouldn't have two variables to refer to the same thing. `GlobalConfiguration` or `GC` but not both. Also we should be consistent in all files.

Reply:
We don't need the first one. Not sure, how it came there. Maybe I missed it mistakenly.